bin/console-conf-wrapper

```diff
@@ -21,6 +21,9 @@ fi
 
 if grep -q 'snapd_recovery_mode=install' /proc/cmdline ; then
     echo "Installing the system, please wait for reboot"
+    snap watch --last=install-system || true
+    snap change --last=install-system || true
+    journalctl -u snapd.service -f
     sleep infinity
```

Comment:
Why `snap change`? previously we had `snap tasks`.

Also, errors are now ignored. This means if getty spawns before install-system task is started, this will not display anything. Or will it? Is there something in place already that ensures that "install-system" change-id has started?

Reply:
Thanks, those are excellent points. I think for beta the easiest path forward is to display the journal output from snapd. I removed the "snap watch/change" now. Especially with the recent changes in snapd install logging this becomes much more useful than it was before. Something we can reconsider after beta, i.e. if we should show some sort of progress here but that's a larger change.